server/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "gitgud-server",
+  "version": "0.1.0",
+  "dependencies": {
+    "body-parser": "^1.18.3",
+    "express": "^4.16.4",
+    "morgan": "^1.9.1"
```

Comment:
Delete the dependencies in the root folder's package.json since you have these same dependencies here.